shiny/Data-Ingest/ui_files/ingest_workflow_ui.R

```diff
@@ -77,37 +78,68 @@ box(
     )
     ),
   hr(),
+splitLayout(
   dateInput(
     "InputStartDate",
     label = "Start Date",
     format = "yyyy-mm-dd",
     startview = "decade"
   ),
   shinyTime::timeInput("StartTimeInput",
-                       label = "Start Time (Hours - Minutes)",
+                       label = "Start Time (HH-MM)",
                        seconds = FALSE),
+  textInput("Timezone",
+            label = "Timezone",
+            placeholder = "")
+  ),
+  splitLayout(
   dateInput(
     'InputEndDate',
     label = 'End Date',
     format = 'yyyy-mm-dd',
     startview = 'decade'
   ),
   shinyTime::timeInput("EndTimeInput",
-                       label = "End Time (Hours-Minutes)",
-                       seconds = FALSE),
-  textInput("Timezone",
-            label = "Timezone (UTC)",
-            placeholder = "UTC +/-"),
+                       label = "End Time (HH-MM)",
+                       seconds = FALSE)
+  ),
   hr(),
   textAreaInput("InputNotes",
                 label = "Notes",
-                height = '100px'),
+                height = '75px'),
   actionButton("createInput", label = "Create Input"),
   actionButton("testBety", label = "Test Bety"),
   p("* Denotes a Required Field"),
   hr(),
   verbatimTextOutput("summInputs"),
   verbatimTextOutput("input_record_df")
+  ),
+## 4. Formats-Variables
+  box(title = h2("3. Formats-Variables"), width = 4, solidHeader = TRUE, status = "success", collapsible = TRUE, collapsed = FALSE,
+      div(id = "add_var_action_button", 
+          actionButton("add_variable", label = "Add New Variable")
+          ),
+      shinyjs::hidden(
+        div(id = "formats_vars_inputs",
+        selectizeInput("pecan_var", choices = NULL, label = "Variable",
+                       options = list(
+                         placeholder = 'Please search or select a site below',
```

Comment:
why are you searching/selecting sites under the variable search?

Reply:
Good catch. This placeholder was copied and pasted from other examples. I'll fix this and add screenshots.